src/Components/MyBooks/MyBooks.tsx

```diff
@@ -135,10 +135,11 @@ const MyBooks = () => {
           availability={availability}
           unavailable={unavailable}
           pending={pending}
+          borrowerUsername={book.borrower?.userName}
           deleteSelectedBook={deleteSelectedBook}
           returnSelectedBook={returnSelectedBook}
-          emailAddress={book.borrower?.emailAddress || ''}
-          location={book.borrower?.location || ''}
+          emailAddress={book.borrower?.emailAddress}
```

Comment:
Nice find. this was probably from when we merged the typescript branch. This is what our Cypress testing should be catching! :(